lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -48,9 +48,27 @@ class HttpApiEvents {
       },
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8021
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'httpApi', {
-      anyOf: [{ type: 'string' }, { type: 'object' }],
+      anyOf: [
+        { type: 'string' },
+        {
+          type: 'object',
+          properties: {
+            method: { type: 'string' },
```

Comment:
Let's define pattern for it (I believe we should go with [`regexp`](https://github.com/ajv-validator/ajv-keywords#regexp) setting to ensure case insensitivity)